Concepts/Makefile

```diff
@@ -0,0 +1,10 @@
+default: concepts emulate introspect
+
+concepts: concepts.cc
+	clang++ -std=c++14  -g -Wall -Wdeprecated concepts.cc -o concepts
```

Comment:
You should probably always have `-Wall -Wextra -pedantic` as the defaults. `-Wall` is really not "all", heck not even `-Wextra` enables really all checks, as you can see with `-Wdeprecated`.